lib/feature_management.rb

```diff
@@ -174,4 +174,13 @@ def self.idv_by_mail_only?
       outage_status.any_phone_vendor_outage? ||
       outage_status.phone_finder_outage?
   end
+
+  # int is our sandbox environment that all partners use to test their apps,
+  # and that the Partnerships team uses for testing and live demos for our partners.
+  # This feature allows pending IPP enrollments to be approved immediately in int,
+  # as opposed to having to wait more than 1 hour, which is not acceptable for
+  # testing purposes. See test/ipp_controller.rb
+  def self.allow_ipp_enrollment_approval?
+    Identity::Hostdata.env == 'int' || Rails.env.development?
```

Comment:
I think this should be a discrete config element so that it could be enabled in `dev`, personal environments, etc.